backend/sse/src/server.ts

```
@@ -0,0 +1,197 @@
+    console.error(err);
+});
+
+const app = express();
+const PORT = 4000;
+
+const allowedOrigins = [
+    "http://localhost:3000",
+];
+
+app.use(cors({
+    origin: (origin: string | undefined, callback: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)